packages/office-addin-test-server/test/test.ts

```diff
@@ -14,6 +14,13 @@ const testValue: string = "Office-Addin-Test-Infrastructure";
 const testValues: any = [];
 
```

Comment:
Are these the only tests that need to be disabled for Linux?  I saw these tests fail in the CI loop but assumed there would be more

Reply:
I've now updated so that the tests that are not supported on Linux are skipped. There is a build error for Windows that I still need to resolve.